js/date-picker/views/lumx.date_picker.html

```diff
@@ -11,47 +11,58 @@
+                <em ng-class="{ 'lx-date-picker__opacity-full': isYearSelected }"><a ng-click="displayYear(true)">{{ now.format('YYYY') }}</a></em>
             </div>
         </div>
 
-        <div class="lx-date-picker__nav">
-            <button class="btn btn--xs btn--teal btn--icon" lx-ripple ng-click="previousMonth()">
-                <i class="mdi mdi--chevron-left"></i>
-            </button>
+        <div ng-init="isYearSelected=false" ng-show="!isYearSelected">
+            <div class="lx-date-picker__nav">
+                <button class="btn btn--xs btn--teal btn--icon" lx-ripple ng-click="previousMonth()">
+                    <i class="mdi mdi--chevron-left"></i>
+                </button>
 
-            <span>{{ month.format('MMMM YYYY') }}</span>
+                <span>{{ month.format('MMMM YYYY') }}</span>
 
-            <button class="btn btn--xs btn--teal btn--icon" lx-ripple ng-click="nextMonth()">
-                <i class="mdi mdi--chevron-right"></i>
-            </button>
-        </div>
+                <button class="btn btn--xs btn--teal btn--icon" lx-ripple ng-click="nextMonth()">
+                    <i class="mdi mdi--chevron-right"></i>
+                </button>
+            </div>
 
-        <div class="lx-date-picker__days-of-week">
-            <span ng-repeat="day in daysOfWeek">{{ day }}</span>
-        </div>
+            <div class="lx-date-picker__days-of-week">
+                <span ng-repeat="day in daysOfWeek">{{ day }}</span>
+            </div>
+
+            <div class="lx-date-picker__days">
+                <span class="lx-date-picker__day lx-date-picker__day--is-empty"
+                      ng-repeat="x in emptyFirstDays">&nbsp;</span><!--
 
-        <div class="lx-date-picker__days">
-            <span class="lx-date-picker__day lx-date-picker__day--is-empty"
-                  ng-repeat="x in emptyFirstDays">&nbsp;</span><!--
+             --><div class="lx-date-picker__day"
+                     ng-class="{ 'lx-date-picker__day--is-selected': day.selected,
+                                 'lx-date-picker__day--is-today': day.today }"
+                     ng-repeat="day in days">
+                    <a ng-click="select(day)">{{ day ? day.format('D') : '' }}</a>
+                </div><!--
 
-         --><div class="lx-date-picker__day"
-                 ng-class="{ 'lx-date-picker__day--is-selected': day.selected,
-                             'lx-date-picker__day--is-today': day.today }"
-                 ng-repeat="day in days">
-                <a ng-click="select(day)">{{ day ? day.format('D') : '' }}</a>
-            </div><!--
+             --><span class="lx-date-picker__day lx-date-picker__day--is-empty"
+                      ng-repeat="x in emptyLastDays">&nbsp;</span>
+            </div>
+        </div>
+
+        <div ng-show="isYearSelected" class="lx-date-picker__years--container">
```

Comment:
`container` is not a modifier, so it should not be after a `--`. I'll let @matmkian confirm it.